kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/core/config/Defaults.kt

```diff
@@ -35,6 +39,7 @@ object Defaults {
    val severity: TestCaseSeverityLevel = TestCaseSeverityLevel.NORMAL
 
    const val coroutineDebugProbes: Boolean = false
+//   const val testCoroutineDispatcher: Boolean = false
```

Comment:
Intentional?

Reply:
Yes, note the PR comments that this is still tbd while I think about it. I won't merge until it's in a final state.